delivery_driver_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+
+from odoo import api, fields, models
+
+
+class StockPickingBatch(models.Model):
+    _inherit = "stock.picking.batch"
+
+    driver_ids = fields.Many2many(
+        "res.partner", string="Drivers", compute="_compute_driver_ids", store=True
```

Comment:
Add the same domain as the stock.picking

```suggestion
        "res.partner", string="Drivers", compute="_compute_driver_ids", store=True, domain="[('is_company', '=', False)]",
```

Reply:
Done. Thanks.